tests/test_pipeline.sh

```diff
@@ -49,7 +49,7 @@ cat << EOT > "${SCRIPT_DIR}/resources/expected_remap.vcf"
 #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	HG001
 chr2	48	.	C	A,T	50	PASS	COMMENT=NANANANANANANANANANANANANAN%ANANANANANANANANANANANANANANANANAN|ANANANANANANA&NANAN ANA\$NANANANANANANANANANANANANANANANANAN^ANANANANANANANANANANANANANANANANANANANANANANANANANANANANANANANANANA NANANANANANANANANANANANANABATMAN;st=+	GT:GQ	1/1:0
 chr2	98	.	C	CG	50	PASS	st=+	GT:GQ	1/1:0
-chr2	1078	.	A	G	50	PASS	st=+;rac=G-A	GT	0/0
+chr2	1078	.	A	G	50	PASS	st=+;rac=chr2|1078|G|A	GT	0/0
```

Comment:
As discussed in the DEV meeting, we might want to confirm that this will pass vcf-validator...

Reply:
I tested it and it does pass.